inlong-audit/audit-service/src/main/java/elector/api/ElectorConfig.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package elector.api;
+
+import lombok.Data;
+import org.apache.commons.lang3.StringUtils;
+
+import java.net.InetAddress;
+
+/**
+ * Elector config
+ */
+@Data
+public class ElectorConfig {
+
+    public static final String MONITOR_COMMON_NAME = "audit";
+    private final String serviceId;
+    private final String leaderId;
+    private String defaultLeaderId;
+    private boolean useDefaultLeader = false;
```

Comment:
How to update “useDefaultLeader”？

Reply:
It can use setUseDefaultLeader() function to update